docs/reference/server/auth/keycloak.rst

```diff
@@ -24,9 +24,40 @@ Interaction schema
             title DummyAuthProvider
             participant "Client"
             participant "Backend"
+            participant "Keycloak"
+
+            == Client Authentication at Keycloak ==
+
+            Client -> Backend : Request endpoint with authentication (/v1/locations)
+
+            Backend x-[#red]> Client: 401 with redirect url in 'details' response field
+
+            Client -> Keycloak : Redirect user to Keycloak login page
+
+            alt Successful login
+                Client --> Keycloak : Log in with login and password
+            else Login failed
+                Keycloak x-[#red]> Client -- : Display error (401 Unauthorized)
+            end
+
+            Keycloak -> Client : Callback to Client /callback which is proxy between Keycloak and Backend
```

Comment:
Client is generic term, and it may not have any endpoints. It's either `frontend` or just mention KeycloakAuthProviderSettings.redirect_url